src/pages/directions.astro

```diff
@@ -0,0 +1,57 @@
+---
+import Layout from "@src/layouts/Layout.astro"
+---
+<Layout title="Directions">
+    <section>
+      <h1>Directions for Geekcamp 2024</h1>
+      Geekcamp 2024 is at <b>SMU Connexion, Level 5</b>. 
+      <h3>
+        1. City Hall (Exit A) - step-free access
+      </h3>
+      <p>
+        Get up to street level via Exit A. You can take the escalator or a lift. Exit Raffles City by the door on the left side. 
+        <img src="/src/data/images/directions2024/cityhall/raffles-city.jpg"/>
```

Comment:
```suggestion
        <Image src="/data/images/directions2024/cityhall/raffles-city.jpg"/>
```

I think it should be this

Reply:
https://docs.astro.build/en/guides/images/

Reply:
thanks Ambrose 😄  I've updated the images to use the astro Image component in line with the docs you sent and also this: https://docs.astro.build/en/reference/modules/astro-assets/#image-properties (for the Image component, alt-text is also required)